### PR TITLE
Add Kustomize + Helm custom tool example

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,4 @@ to explore ArgoCD and GitOps!
 | [helm-dependency](helm-dependency/) | Demonstrates how to customize an OTS (off-the-shelf) helm chart from an upstream repo |
 | [blue-green-deploy](blue-green-deploy/) | Demonstrates an Argo CD Sync hook which performs a blue/green deployment |
 | [sock-shop](sock-shop/) | A microservices demo application (https://microservices-demo.github.io) |
+| [kustomized-helm](kustomized-helm/) | Demonstrates an application comprised of a helm chart and customized using `kustomize` |
```
Comment (alexec) [inline README.md]: ` kustomized-helm` -->  `customized-templating-tool`? confusing? makes me think of kustomize, or helm

Comment (limnick) [reply README.md]: yeah I would also suggest framing this as a way to use any user specified template tool, it just happens to use kustomize in this example

Comment (alexmt) [reply README.md]: Agree. Added one more example which uses [Kasane](https://github.com/google/kasane) and moved both into `/plugins` directory. @alexec , @limnick do you think it looks better now?

Comment (limnick) [reply README.md]: looks good to me! Thanks for the changes!

I haven't tried running through the steps myself to confirm they work as written.

**Comment by limnick** [comment]
@alexmt noted on slack that this PR depends on https://github.com/argoproj/argo-cd/pull/1174 being merged first to change `path` to `command` in the `configManagementPlugins` argo-cd configmap

**Pull request merged**